tutorials/llm/llama-3/slimpajama/README.md

```diff
@@ -0,0 +1,48 @@
+**Introduction**
+
+Welcome to the NeMo SlimPajama Data Pipeline and Pretraining tutorial! This tutorial provides a step-by-step guide to preprocessing the SlimPajama dataset and pretraining a Llama based model using the NeMo 2.0 library.
+
+The tutorial includes two Jupyter notebooks: `data_pipeline.ipynb` and `pretraining.ipynb`. The `data_pipeline.ipynb` notebook provides a data pipeline to preprocess the SlimPajama dataset, including downloading, extracting, concatenating and tokenizing the data. The `pretraining.ipynb` notebook provides a pretraining recipe to train a language model using the preprocessed data.
+
+This repository is designed to be used with the NeMo 2.0 and NeMo-Run.
+
+**Pre-requisites / Requirements**
+
+- System Configuration
+  - For Preprocessing: access to any CPU node should be sufficient. Please reach out to us if you run into errors.
+  - For Pretraining: access to at least 1 NVIDIA GPUs with a cumulative memory of at least 48GB.
+  - A Docker-enabled environment, with NVIDIA Container Runtime installed, which will make the container GPU-aware.
+- Software Requirements
+  - Use the latest [NeMo Framework Container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo/tags) . Note that you must be logged in to the container registry to view this page.
+  - This notebook uses the container: nvcr.io/nvidia/nemo:dev.
+  - Get your Hugging Face [access token](https://huggingface.co/docs/hub/en/security-tokens), which will be used to download assets from Hugging Face.
+  - Download Jupyter Lab or Jupyter Notebook in your environment if not already installed.
+- NeMo 2.0 and NeMo-Run
+  - We will use NeMo 2.0 and NeMo-Run for this tutorial. Both are already available in the NeMo Framework Container.
+
+**Getting started**
+
+Assuming you have all the pre-requisites installed, you can get started by following these steps:
+1. Start and enter the dev container by running:
+   ```bash
+    docker run \
+    --gpus device=1 \
```

Comment:
Sorry I copy pasted this line wrong in the nemo2-sft-peft first. I changed it to `all` later. Even if user want to use 1 GPU it's safer to assume device=0 instead of 1. (customer ran into this and asked for fix...)